blocks/block-edit/index.js

```diff
@@ -26,6 +27,8 @@ export function BlockEdit( props ) {
 		return null;
 	}
 
+	const { autocompleters = [] } = blockType;
```

Comment:
For blocks which don't define their own `autocompleters`, this assign a new reference value every single render, which I assume will then trickle down to `AutocompletionContext.Provider` as a new value (and update its context accordingly).

Could we...

- Gracefully handle an undefined `autocompleters` in the provider component?
- Move the default value up to the module scope so it's a shared value.